src/templates/match-approved-email.html

```diff
@@ -25,11 +25,11 @@
 <p></p>
 The exchange rate is currently 1 {%CURRENCY_A%} = {%ONE_CURRENCY_A_AS_B%} {%CURRENCY_B%}. 
 <p></p>
-<p>So {%TO_CHARITY_A%} {%CURRENCY_A%}={%TO_CHARITY_B%} {%CURRENCY_B%}, and both charities receive the same amount of money.</p>
```

Comment:
i _think_ `{%TO_CHARITY_A%} {%CURRENCY_A%}={%TO_CHARITY_B%} {%CURRENCY_B%}` was correct, but it shoudl probably be tested